ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewCellDemoController.swift

```diff
@@ -122,18 +122,20 @@ extension TableViewCellDemoController {
         let section = sections[indexPath.section]
         let item = section.item
         if section.title == "Inverted double line cell" {
-            cell.setup(
-                attributedTitle: NSAttributedString(string: item.text1,
-                                                    attributes: [.font: TextStyle.footnote.font,
-                                                                 .foregroundColor: Colors.Table.Cell.footer]),
-                attributedSubtitle: NSAttributedString(string: item.text2,
-                                                       attributes: [.font: TextStyle.body.font,
-                                                                    .foregroundColor: Colors.Table.Cell.title]),
-                footer: TableViewCellSampleData.hasFullLengthLabelAccessoryView(at: indexPath) ? "" : item.text3,
-                customView: TableViewSampleData.createCustomView(imageName: item.image),
-                customAccessoryView: section.hasAccessory ? TableViewCellSampleData.customAccessoryView : nil,
-                accessoryType: TableViewCellSampleData.accessoryType(for: indexPath)
-            )
+            DispatchQueue.main.async {
```

Comment:
Was this a bug in only the demo, and not a bug with the actual cell?

Reply:
I don't believe so - the attributes are all loaded correctly, but they are only partially updating. The cells display correctly after scrolling, which is why I interpreted as a demo controller issue

Reply:
Updating this thread, it's an issue with the Label component, so I'll be opening up a new PR for that so I can also merge into fluent2-tokens!